File: Iniciante/1043 - Triângulo/Python/1043.py
def main():
    a, b, c = map(float, input().split())
    is_triangle = check_is_triangle(a, b, c)
    if is_triangle:
        perimeter = calculate_perimeter(a, b, c)
        print(f"Perimetro = {perimeter:.1f}")
    else:
        area = calculate_area(a, b, c)
        print(f"Area = {area:.1f}")


def check_is_triangle(a, b, c):
    return a + b > c and a + c > b and b + c > a


def calculate_perimeter(a, b, c):
    return a + b + c


def calculate_area(a, b, c):
    return ((a + b) * c) / 2.0


if __name__ == "__main__":
    main()

File: Iniciante/1043 - Triângulo/JavaScript/1043.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const [a, b, c] = input.trim().split(/\s+/).map(Number);
  const isTriangle = checkIsTriangle(a, b, c);
  if (isTriangle) {
    const perimeter = calculatePerimeter(a, b, c);
    console.log(`Perimetro = ${perimeter.toFixed(1)}`);
  } else {
    const area = calculateArea(a, b, c);
    console.log(`Area = ${area.toFixed(1)}`);
  }
}

function checkIsTriangle(a, b, c) {
  return a + b > c && a + c > b && b + c > a;
}

function calculatePerimeter(a, b, c) {
  return a + b + c;
}

function calculateArea(a, b, c) {
  return ((a + b) * c) / 2.0;
}

main();
